controllers/service/service_controller.go

```diff
@@ -10,7 +10,7 @@ import (
 	"sigs.k8s.io/aws-load-balancer-controller/controllers/service/eventhandlers"
 	"sigs.k8s.io/aws-load-balancer-controller/pkg/annotations"
 	"sigs.k8s.io/aws-load-balancer-controller/pkg/aws"
-	"sigs.k8s.io/aws-load-balancer-controller/pkg/config"
+	cfg "sigs.k8s.io/aws-load-balancer-controller/pkg/config"
```

Comment:
any reason to use named import for this package?

Reply:
Due to the conflict mentioned in https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/2587#discussion_r864474728.